xbmc/addons/Skin.cpp

```diff
@@ -369,6 +369,8 @@ const INFO::CSkinVariableString* CSkinInfo::CreateSkinVariable(const std::string
 
 void CSkinInfo::OnPreInstall()
 {
+  if (IsInUse())
+    CApplicationMessenger::GetInstance().SendMsg(TMSG_EXECUTE_BUILT_IN, -1, -1, nullptr, "UnloadSkin");
```

Comment:
Yes, *on windows*:) This is a windows design issue because there is no way for us to possibly ensure that all processes close the file before we move it.